Formula/mapnik.rb

```diff
@@ -47,55 +48,50 @@ class Mapnik < Formula
   depends_on "webp"
 
   def install
-    ENV.cxx11
-    ENV["PYTHON"] = "python3.9"
-
-    # Work around "error: no member named 'signbit' in the global namespace"
-    # encountered when trying to detect boost regex in configure
-    ENV.delete("SDKROOT") if DevelopmentTools.clang_build_version >= 900
-
-    boost = Formula["boost"].opt_prefix
-    freetype = Formula["freetype"].opt_prefix
-    harfbuzz = Formula["harfbuzz"].opt_prefix
-    icu = Formula["icu4c"].opt_prefix
-    jpeg = Formula["jpeg-turbo"].opt_prefix
-    libpng = Formula["libpng"].opt_prefix
-    libtiff = Formula["libtiff"].opt_prefix
-    proj = Formula["proj"].opt_prefix
-    webp = Formula["webp"].opt_prefix
+    boost = Formula["boost"]
+    harfbuzz = Formula["harfbuzz"]
+    icu = Formula["icu4c"]
+    jpeg = Formula["jpeg-turbo"]
+    libpng = Formula["libpng"]
+    libtiff = Formula["libtiff"]
+    proj = Formula["proj"]
+    webp = Formula["webp"]
 
     args = %W[
       CC=#{ENV.cc}
       CXX=#{ENV.cxx}
```

Comment:
Ah. Issue is directly depending on `scons` causes `brew` to use `--env=std`. May not be worth the hassle to do this and instead just wait for next release with `cmake` support.